bing.com.BingWebMasterDPOO.json

```diff
@@ -0,0 +1,22 @@
+{
+  "providerId": "bing.com",
```

Comment:
This template as defined does not require signatures.  I'm not sure what the fishing thread here is; could I send a phishing mail out with my GUID, get the domain owner to click my link and validate me, and then have access to domain level functionality?  Read section 4.2.3 of the spec for more information.  I'm guessing you'll want to either sign your requests, or at least turn on warnPhising.....

Reply:
Agree. Resolved